test/close-pipelining.test.js

```diff
@@ -36,39 +36,9 @@ test('Should return 503 while closing - pipelining', async t => {
   await instance.close()
 })
 
-const isNodeVersionGte1819 = semver.gte(process.version, '18.19.0')
-test('Should not return 503 while closing - pipelining - return503OnClosing: false, skip Node >= v18.19.x', { skip: isNodeVersionGte1819 }, async t => {
-  const fastify = Fastify({
-    return503OnClosing: false,
-    forceCloseConnections: false
-  })
-
-  fastify.get('/', (req, reply) => {
-    fastify.close()
-    reply.send({ hello: 'world' })
-  })
-
-  await fastify.listen({ port: 0 })
-
-  const instance = new Client('http://localhost:' + fastify.server.address().port, {
-    pipelining: 2
-  })
-
-  const codes = [200, 200, 200]
-  const responses = await Promise.all([
-    instance.request({ path: '/', method: 'GET' }),
-    instance.request({ path: '/', method: 'GET' }),
-    instance.request({ path: '/', method: 'GET' })
-  ])
-  const actual = responses.map(r => r.statusCode)
-
-  t.same(actual, codes)
-
-  await instance.close()
-})
-
-test('Should close the socket abruptly - pipelining - return503OnClosing: false, skip Node < v18.19.x', { skip: !isNodeVersionGte1819 }, async t => {
-  // Since Node v18, we will always invoke server.closeIdleConnections()
+const isNodeVersionGte19 = semver.gte(process.version, '19.0.0')
+test('Should close the socket abruptly - pipelining - return503OnClosing: false, skip Node < v19', { skip: !isNodeVersionGte19 }, async t => {
```

Comment:
```suggestion
test('Should close the socket abruptly - pipelining - return503OnClosing: false', async t => {
```

You can remove the `skip` condition.